docs/deployments/exporting-users.mdx

```diff
@@ -3,9 +3,9 @@ title: Exporting Users
 description: Learn how to export user's data from your Clerk application.
 ---
 
-## Exporting your user's data from Clerk
+# Exporting your user's data from Clerk
```

Comment:
[avoid use of gerunds](https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#24-avoid-gerunds--ing-words)

```suggestion
# Export your user's data from Clerk
```